### PR TITLE
[codesign] remove password files after runs

### PR DESCRIPTION
**Pull request by XilaiZhang**
fixes https://github.com/flutter/flutter/issues/115117.

Ensure the password files are cleaned up after runs.

### CODE REVIEW

File: codesign/test/file_codesign_visitor_test.dart [hunk 1]
```diff
@@ -1200,6 +1204,10 @@ status: Invalid''',
         ),
       );
       rootDirectory = fileSystem.systemTempDirectory.createTempSync('conductor_codesign');
+
+      await fileSystem.file(appSpecificPasswordFilePath).delete();
```
Comment (CaseyHillers) [inline codesign/test/file_codesign_visitor_test.dart:1208]: L24 should be made:

```
  late file FileSystem fileSystem;
```

Then the `setUp` should instantiate the filesystem.

Comment (XilaiZhang) [reply codesign/test/file_codesign_visitor_test.dart]: Done. Yeah this is cleaner. By the way would recreating filesystems be cheaper than recreating files?

File: codesign/test/file_codesign_visitor_test.dart [hunk 2]
```diff
@@ -21,16 +21,17 @@ void main() {
...
+  late MemoryFileSystem fileSystem;
   late FakeProcessManager processManager;
   late cs.FileCodesignVisitor codesignVisitor;
-
-  Directory rootDirectory = fileSystem.systemTempDirectory.createTempSync('conductor_codesign');
+  late Directory rootDirectory;
 
   group('test reading in passwords: ', () {
     setUp(() {
+      fileSystem = MemoryFileSystem.test();
```
Comment (CaseyHillers) [inline codesign/test/file_codesign_visitor_test.dart:33]: This doesn't block your PR, but in the future you can remove a lot of this duplicate work in a refactor by nesting your setUps.

It's common to have a group for the entire file, then groups for different parts of logic

**Review by CaseyHillers — APPROVED** [review]

File: codesign/test/file_codesign_visitor_test.dart (hunk 2 above)
Comment (XilaiZhang) [reply codesign/test/file_codesign_visitor_test.dart]: Thanks a lot for the suggestion! Refactored

**Pull request merged**